start-nodemanagers.sh

```diff
@@ -31,6 +31,10 @@ do
     then
         echo "Kill screen sessions on" ${HOSTNAME}
         ssh ${USER}@${HOSTNAME} "killall screen"
+    elif  [ ${MODE} = "insert" ]
+    then
+        echo "Insert nodemanager" ${HOSTNAME} "in database"
+        ssh ${USER}@${HOSTNAME}  "bash ${SCRIPT_DIR}/insert-nodemanager.sh"
```

Comment:
How do you start the nodemanager now? Do you have to run `start-nodemanagers.sh insert` and then `start-nodemanagers.sh start`?

Reply:
Why this has to happen on the remote node? Why not just locally?

Reply:
You could also put this in the "start" section, just running the insertion locally on the node where you start the request.

Reply:
First `start` mode, then `insert` mode. It needs to happen remotely because otherwise `os.cpu_count()` will always return the cpu count from `portal1` rather than any of the nodemanager machines. 
I could put it in the `start` section, but at least after some `sleep x` seconds because it takes some time to start the nodemanager daemon and we need information from the `rm.get_resources(node)` function which becomes available on when `node` is actually connected to `rm`

Reply:
ah, i see! i missed that you add the total count as well, i thought you put only the name and 0.